packages/zk-certificates/test/contracts/BasicKYCExampleDAppTest.ts

```diff
@@ -72,8 +72,9 @@ describe('BasicKYCExampleDApp', () => {
       'VerificationSBT',
       deployer,
     );
-    verificationSBT =
-      (await verificationSBTFactory.deploy()) as VerificationSBT;
+    verificationSBT = (await verificationSBTFactory.deploy(
+      'test URI',
```

Comment:
This is not a valid URI. I think it would be good if we upload a static json metadata file, for example on Github pages. Otherwise the SBT would not be properly displayed in wallets and explorers looking for the standardized metadata: https://docs.opensea.io/docs/metadata-standards#metadata-structure

Reply:
In the unittest we can probably ignore it, but adding it to the deployment scripts would be important.